src/mono/mono/metadata/class-setup-vtable.c

```diff
@@ -1065,7 +1065,7 @@ handle_dim_conflicts (MonoMethod **vtable, MonoClass *klass, GHashTable *conflic
 		int nentries = 0;
 		MonoMethod *impl = NULL;
 		for (l = entries; l; l = l->next) {
-			if (l->data) {
+			if (l->data && l->data != impl) {
 				nentries ++;
```

Comment:
Is this part of the PR ?

Reply:
looks like rebase leftover

Reply:
it was actually a bug that I forgot that I fixed